### ISSUE TITLE
Weird Bar in new Tab

### ISSUE THREAD

**Issue by Qliphort8** [zen-browser/desktop#3533]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

when i open a new tab there is this line underneath my adress bar that i cant get rid of
![Desktop Screenshot 2024 12 11 - 01 48 16 88](https://github.com/user-attachments/assets/911eb607-988c-45a6-a0a8-b79d44a238e2)

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.0 (Firefox 133.0) (64-bit)

### What platform are you seeing the problem on?

Windows

**Comment by gnc-07** [comment]
When you exit the customization mode is that dotted line still there? Also, to make sure I'm understanding you correctly, by line, you are referring to the dotted line forming a rectangle in your screenshot, correct?

**Comment by Qliphort8** [comment]
> When you exit the customization mode is that dotted line still there? Also, to make sure I'm understanding you correctly, by line, you are referring to the dotted line forming a rectangle in your screenshot, correct?

Yes exactly, it's not dotted anymore but the bar is still there, I cannot remove it in the customization.

**Comment by Qliphort8** [comment]
i reinstalled it
